source/install/ee-prod-rhel-6.rst

```diff
@@ -323,7 +323,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
+4. Update **Notification** > **Email** settings. We recommend using an email sending service.
```

Comment:
The second sentence seems weird--are we giving and instruction or a recommendation? 

Propose instead `Update **Notification** > **Email** settings to setup an SMTP email service.`